model/BookingInfoRequest.ts

```diff
@@ -2,11 +2,9 @@ import { FiveFieldsBookingInfoRequest } from "./FiveFieldsBookingInfoRequest"
 import { FlightBookingInfoRequest } from "./FlightBookingInfoRequest"
 import { HotelBookingInfoRequest } from "./HotelBookingInfoRequest"
 import { InvoiceBookingInfoRequest } from "./InvoiceBookingInfoRequest"
-import { LegacyBookingInfoRequest } from "./LegacyBookingInfoRequest"
```

Comment:
is `LegacyBookingInfoRequest` not used anywhere?
in that case it should be removed